src/main/webapp/definitions/alfresco-content-models.yaml

```diff
@@ -0,0 +1,738 @@
+        '400':
+          description: |
+            Invalid parameter: **modelId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to retrieve **modelId**
+        '404':
+          description: |
+            **modelId** does not exist
+  '/models/{modelId}/types':
+    get:
+      x-alfresco-since: "6.2.2"
+      tags:
+        - models
+      summary: List types for the model
+      description: |
+        **Note:** this endpoint is available in Alfresco 6.2.2 and newer versions.
+
+        Get the list of types defined in the model **modelId**.
+
+        The default sort order is **name** ascending, but you can use an optional **ASC** or **DESC** 
+        modifier to specify an ascending or descending sort order. 
+
+        For example, specifying ```orderBy=name DESC``` returns QName entries in descending **name** order.
+        You can specify one or more of the following fields in the **orderBy** parameter:
+        * name
+        * namespaceUri
+        * namespacePrefix  
+      operationId: listModelTypes
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/modelIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/orderByParam'
+      responses:
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/QNameEntry'
+        '400':
+          description: |
+            Invalid parameter: value of **maxItems**, **skipCount**, **orderBy** is invalid or **modelId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to retrieve **modelId**
+        '404':
+          description: |
+            **modelId** does not exist
+  '/models/{modelId}/types/{typeName}':
+    get:
+      x-alfresco-since: "6.2.2"
+      tags:
+        - models
+      summary: Get a type
+      description: |
+        **Note:** this endpoint is available in Alfresco 6.2.2 and newer versions.
+
+        Get information about the type **typeName** defined in **modelId**
+      operationId: getType
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/modelIdParam'
+        - $ref: '#/parameters/typeNameParam'
+      responses:
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/TypeEntry'
+        '400':
+          description: |
+            Invalid parameter: **modelName** or **typeName** is not a valid format
```

Comment:
Make sure documentation matches parameter names, should be **modelId** and **typeId** (when changed elsewhere). There are a couple more occurrences below too.